packages/block-library/src/media-text/style.scss

```diff
@@ -1,8 +1,6 @@
-.wp-block-media-text {
-	display: grid;
-}
 
 .wp-block-media-text {
+	display: grid;
```

Comment:
Can I confirm this is to allow the alignment to work on this item?

Reply:
It's just moving the property from the duplicate selector above.